types/relay-runtime/tslint.json

```diff
@@ -2,6 +2,8 @@
     "extends": "dtslint/dt.json",
     "rules": {
         // We have some internal types that are helpfully tagged with @private
-        "no-redundant-jsdoc-2": false
+        "no-redundant-jsdoc-2": false,
+        "no-unnecessary-generics": false,
```

Comment:
this and the below reflected existing failures; not sure how they didn't come up before

Reply:
Perhaps an older version of tslint was used previously?  🤷‍♂